### ISSUE TITLE
Markdown blockquote shouldn't require a leading blank line

### ISSUE THREAD

**Issue by westy92** [jgm/pandoc#7069]
Source (markdown):
```markdown
This is some text.
> This is a blockquote
```

Expected HTML output:
```html
<p>This is some text.</p>
<blockquote>
<p>This is a blockquote</p>
</blockquote>
```

[Actual HTML output:](https://pandoc.org/try/?text=This+is+some+text.%0A%3E+This+is+a+blockquote&from=markdown&to=html5&standalone=0)
```html
<p>This is some text. &gt; This is a blockquote</p>
```

Verison info:
```shell
$ pandoc --version
pandoc.exe 2.11.4
Compiled with pandoc-types 1.22, texmath 0.12.1, skylighting 0.10.2,
citeproc 0.3.0.5, ipynb 0.1.0.1
```

**Comment by jgm** [comment]
From the manual:

#### Extension: `blank_before_blockquote` ####
                                                      
Standard Markdown syntax does not require a blank line before a         
block quote.  Pandoc does require this (except, of course, at       
the beginning of the document). The reason for the requirement            
is that it is all too easy for a `>` to end up at the beginning           
of a line by accident (perhaps through line wrapping). So,
unless the `markdown_strict` format is used, the following does
not produce a nested block quote in pandoc:      
                               
    > This is a block quote.
    >> Nested.                            
                    

Bottom line:  if you don't want to leave a blank line, use `-f markdown-blank_before_blockquote`, or `-f markdown_strict`, or `-f gfm` or `-f commonmark` etc.  But this is not a bug.

**Comment by jgm** [comment]
If you were aware of this and meant to advocate for a change in the default setting of this extension for pandoc's markdown, then I suggest you do that on the pandoc-discuss mailing list, which is where big changes to longstanding features need to be discussed.

**Comment by westy92** [comment]
`-f markdown-blank_before_blockquote` worked great, thanks!
